modules/medianetRtdProvider.js

```diff
@@ -0,0 +1,112 @@
+import { submodule } from '../src/hook.js';
+import * as utils from '../src/utils.js';
+import { getGlobal } from '../src/prebidGlobal.js';
+import includes from 'core-js-pure/features/array/includes.js';
+
+const MODULE_NAME = 'medianet';
+const SOURCE = MODULE_NAME + 'rtd';
+const AD_UNIT_CODE_TARGETING_KEY = 'mnadc';
+const OPEN_RTB_FIELD = 'ortb2Imp';
+
+const getClientUrl = (customerId, domain) => `https://warp.media.net/js/tag/prebidrtdclient.js?cid=${customerId}&dn=${domain}`;
```

Comment:
please include prominent disclosures that this submodule loads external js and either lock that js to an open source version or make the js location configurable so that the publisher can self host if they so choose.

Reply:
see mass module for an example of external js being configurable

Reply:
Hi @patmmccann, as per [Real Time Data Module Rule No. 1](https://docs.prebid.org/dev-docs/module-rules.html#real-time-data-module-rules) we have added a disclosure in our docs PR [here](https://github.com/prebid/prebid.github.io/pull/3023/files#diff-5c86761f5eb43005a0f07393aa1821d833de2cf1e4544dad4c575a504436f156R28)

We want to keep minimal changes at the publisher's end thus we are not taking the tag URL from publisher